mypyc/test/test_emitmodule.py

```diff
@@ -22,17 +22,10 @@
 class TestCompiler(DataSuite):
     """Test cases that compile to C and perform checks on the C code."""
 
-    def __init__(self, *, update_data: bool) -> None:
-        pass
-
-    @classmethod
-    def cases(cls) -> List[DataDrivenTestCase]:
-        c = []  # type: List[DataDrivenTestCase]
-        for f in files:
-            c += parse_test_cases(
-                os.path.join(test_data_prefix, f),
-                None, test_temp_dir, True)
-        return c
+    files = files
```

Comment:
Since this is repeated here and in the other data suites, perhaps it would make sense to have a base class to reduce repetition?